### DISCUSSION TITLE
Unable to find Hilt

### DISCUSSION THREAD

**Discussion by jakepurple13** [KaustubhPatange/Moviesy#24] [Q&A] [answered]
I'm trying to compile the project but I get stopped by :

The Hilt Android Gradle plugin is applied but no com.google.dagger:hilt-android dependency was found.

I haven't used Hilt before but this is just from importing from version control. Am I missing something that I should have but don't?

Also not sure if this should be a discussion or an issue since its on the compiling side and not the code side.

**Comment by KaustubhPatange** [comment, marked as answer]
Did you follow this compilation [guide](https://github.com/KaustubhPatange/Moviesy/wiki/Compilation-guide)?

**Reply by jakepurple13** [reply]
Yeah I did.

**Reply by KaustubhPatange** [reply]
I just freshly cloned the repository, followed the steps from the [guide](https://github.com/KaustubhPatange/Moviesy/wiki/Compilation-guide) & it worked. Surely you are doing something wrong.

Try it again by freshly cloning the repository again. It should work!

**Reply by jakepurple13** [reply]
I think I got it after a lot of changes. I had to manually find the gradle-wrapper.properties file. Had to remove the hilt and dagger plugins (but still got it to build and run with them commented out). And I also had to but the includes into the settings.gradle.kts file manually. I may have missed one or two more but those are definitely things I did.

**Reply by KaustubhPatange** [reply]
I don't know what exactly is the problem from your side (maybe studio or gradle is messing something up), because the [CI server](https://github.com/KaustubhPatange/Moviesy/actions/workflows/build.yml) are able to build the project successfully & it basically follows the exact guide mentioned above.
